dags/mantaray/run_mantaray_jobs.py

```diff
@@ -38,6 +40,54 @@
     if re.match(pattern, job["task_name"]):
       workload_file_name_list.append(job["file_name"])
 
+
+  def run_test_code_on_persistent_TPUVM():
+    gcloud_command = (
+        f"gcloud compute tpus tpu-vm ssh manfei-2025-v6e-4 --zone=us-east5-b --project=cloud-ml-benchmarking --ssh-flag='-t' --worker=all \
+        --command=\"sudo docker run -it --privileged --net host --shm-size=16G --name testooo docker.io/vllm/vllm-tpu:270a5da495d24e947a71e2fa0c56635f4fad2dc3 \
```

Comment:
Are we always using the same vllm image?

Reply:
thanks, updated to nightly docker image